frontend/src/helpers/form/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './get-options.helper';
+export * from './get-default-option.helper';
+export * from './create-options.helper';
```

Comment:
Helpers should be always in the own-folder
```suggestion
export * from './get-options/get-options.helper';
export * from './get-default-option/get-default-option.helper';
export * from './create-options/create-options.helper';
```